final/index.html

```diff
@@ -40,16 +40,14 @@ <h1 class="header__title">Weather PWA</h1>
           </div>
         </div>
         <div class="description">
-          <div class="feels-like">
```

Comment:
The API didn't have a comparable "feels like" data point, so I used sunrise and sunset instead.